client/client.go

```diff
@@ -58,7 +58,7 @@ import (
 )
 
 // DefaultVersion is the version of the current stable API
-const DefaultVersion string = "1.23"
+const DefaultVersion string = "1.25"
```

Comment:
should we drop one and just use the other instead? (I would remove the `client/client.go` one)